raml-parser-interface/src/main/java/org/mule/apikit/model/parameter/Parameter.java

```diff
@@ -52,4 +52,5 @@ public interface Parameter {
 
   Optional<FileProperties> getFileProperties();
 
+  boolean isNullable();
```

Comment:
I'm wondering if it is worth to add javadoc and `@since` here

Reply:
this requires a minor version, we can put @since

Reply:
Makes sense to me, @camilalrocca lets add a simple javadoc including `@since <next minor version>`